### DISCUSSION TITLE
نحوه استفاده در انگولار

### DISCUSSION THREAD

**Discussion by FarazGolshahii** [majidh1/JalaliDatePicker#118] [Question]
سلام وقتتون بخیر. میخواستم بدونم از این پکیج چطوری میتونم در انگولار استفاده کنم. آیا به مشکل نمیخوره ؟
اگه کسی از دوستان هم این کار رو انجام داده ممنون میشم نمونه مورد استفاده شدشو و طریفه استفاده ازش رو بهم بگه.
ممنون.

**Comment by majidh1** [comment]
سلام وقت بخیر میتونید در لینک زیر اطلاعات پیاده سازی برای هر فریم ورک را مشاهده کنید
https://github.com/majidh1/JalaliDatePicker/blob/main/Sample.md
